packages/react-notifications/src/components/notification-kinds/api-error-message/api-error-message.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { useIntl, IntlShape } from 'react-intl';
+import has from 'lodash/has';
+import { reportErrorToSentry } from '@commercetools-frontend/sentry';
+import { TAppNotificationApiError } from '@commercetools-frontend/constants';
+import messages from './messages';
+
+const regexInvalidOperationRequiredAttribute = /Required attribute '(.*)' cannot be removed/;
+
+type ExtraErrorFields = {
```

Comment:
This type contains all the possible extra fields from any of the API errors that we explicitly handle here